stdlib/public/BackDeployConcurrency/AsyncDropFirstSequence.swift

```diff
@@ -22,12 +22,12 @@ extension AsyncSequence {
   ///
   /// In this example, an asynchronous sequence called `Counter` produces `Int`
   /// values from `1` to `10`. The `dropFirst(_:)` method causes the modified
-  /// sequence to ignore the values `0` through `4`, and instead emit `5` through `10`:
+  /// sequence to ignore the values `1` through `3`, and instead emit `4` through `10`:
```

Comment:
Note here: These numbers need to be fixed.a

Reply:
Yes, the old numbers were wrong. The list starts at 1, not a 0.